litmus-portal/frontend/src/components/DateRangeSelector/index.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Why is this required? I've used the same date range picker earlier, I don't think we need this.

Reply:
It was throwing some error in eslint so i ignored this rule .. @amityt it is needed for eslint checks to pass..